features/step_definitions/menu_steps.rb

```diff
@@ -13,6 +13,12 @@
   end
 end
 
+Then /^I should see there is current API/ do
+  within '#mainmenu' do
+    assert has_css? '.pf-c-nav__section-title'
```

Comment:
I suggest asserting the actual title of the current api. It should be something like:

```suggestion
    assert has_css?('.pf-c-nav__section-title', text: @service.name)
```